lib/linux.js

```diff
@@ -41,18 +41,23 @@ var LinuxBuilder = {
       linux.archName = 'x86_64';
     }
 
-    var tmpFolder = path.join( os.tmpDir(), linux.executable );
+    var outFilename = linux.executable + '-' +
```

Comment:
I'm not a big fan of these long lines... But we can surely decide together. :bowtie: 

As this file was mainly written by @xseignard. What do you think?